mne/viz/topo.py

```diff
@@ -638,23 +685,29 @@ def plot_topo_image_epochs(epochs, layout=None, sigma=0., vmin=None,
     """
     scalings = _handle_default('scalings', scalings)
     data = epochs.get_data()
-    if vmin is None:
-        vmin = data.min()
-    if vmax is None:
-        vmax = data.max()
+    scale_coeffs = list()
+    for idx in range(epochs.info['nchan']):
+        ch_type = channel_type(epochs.info, idx)
+        scale_coeffs.append(scalings[ch_type])
+    for epoch_data in data:
+        epoch_data *= np.asarray(scale_coeffs)[:, np.newaxis]
+    vmin, vmax = _setup_vmin_vmax(data, vmin, vmax)
```

Comment:
This ``vmin, vmax`` thing wasn't really working before. It's still not working great if you have several types of channels in the epoch collection, but I don't want to change the behavior too much. Ideally, I think there should be a ``ch_type`` param for this function.